chainerx_cc/chainerx/routines/math.cc

```diff
@@ -531,6 +562,10 @@ Array Maximum(const Array& x1, Scalar x2) {
 
 Array Maximum(Scalar x1, const Array& x2) { return Maximum(x2, x1); }
 
+Array Maximum(const Array& x1, const Array& x2) {
+    return IfLessElse(x1, x2, x2, x1);  // x1 < x2 ? x2 : x1
```

Comment:
How about writing as `IfGreaterElse(x1, x2, x1, x2)` if you will complete #6541 first?

Reply:
Merged #6541.